src/index.ts

```diff
@@ -74,7 +74,9 @@ function dryRun(
 export async function main(
   args: UploaderArgs,
 ): Promise<void | Record<string, unknown>> {
-
+  
+  console.debug({ args });
+  
```

Comment:
```suggestion
  verbose(`Arguments passed: ${args}`, Boolean(args.verbose))
```

Reply:
That line was mistakenly added. I've since removed it.